js/hoverzoom.js

```diff
@@ -945,6 +945,9 @@ var hoverZoom = {
             if (imgFullSize) {
                 var link = hz.currentLink, data = link.data();
                 if (data.hoverZoomGallerySrc) {
+                    if(data.hoverZoomGallerySrc.length === 1){
```

Comment:
Should merge this "if" with the previous "if" and also respect coding style (whitspaces).